### PR TITLE
Update image repositories

### PR DESCRIPTION
**Pull request by whummer**
Update configuration files to point to public Docker image repository.

### CODE REVIEW

File: templates/services/trainingdata-deployment.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ spec:
 {{ end }}
       containers:
       - name: ffdl-trainingdata-container
-        image: registry.ng.bluemix.net/{{.Values.docker.namespace}}/{{.Values.docker.image_prefix}}metrics:{{.Values.trainingdata.version}}
+        image: docker.io/{{.Values.docker.namespace}}/{{.Values.docker.image_prefix}}metrics:{{.Values.trainingdata.version}}
```
Comment (Tomcli) [inline templates/services/trainingdata-deployment.yml]: How come this is not using `{{.Values.docker.registry}}`?

Comment (whummer) [reply templates/services/trainingdata-deployment.yml]: Thanks, good catch! Will fix in a follow-up PR.

File: values.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ env: dev
 services:
   expose_node_port: true
 docker:
-  registry: registry.ng.bluemix.net
+  registry: docker.io
   namespace: dlaas_dev
```
Comment (Tomcli) [inline values.yaml]: I believe we also need to change the name space to `ffdl`?

Comment (animeshsingh) [reply values.yaml]: Lets do that in a susequent PR - once we have this working here fully

Comment (animeshsingh) [reply values.yaml]: Lets test it on Vagrant/Minikube/IBM Cloud once with this repository - and mark it as an inital release when everything is working - then we move on to adding other things

Comment (Tomcli) [reply values.yaml]: yes

Comment (Tomcli) [reply values.yaml]: Yeah, let's fix this in another PR. I also encountering other issues.

**Pull request merged**